### PR TITLE
Postman

### PR DESCRIPTION
**Pull request by alanlamaswolox**
## Summary

Updated a few methods and models to make requests work

## Trello
[link](https://trello.com/c/XK8yNHA8/39-postman)

### CODE REVIEW

**Review by AleFalcon — DISMISSED** [review]
It's okay

File: src/main/java/wolox/training/models/Book.java [hunk 1]
```diff
@@ -61,7 +63,7 @@
     private String isbn;
 
     @ManyToMany(mappedBy = "books")
-    private List<User> users = new ArrayList<User>();
+    private List<User> user = new ArrayList<User>();
```
Comment (Gianni-Nicolas) [inline src/main/java/wolox/training/models/Book.java]: Why did you change the name of the list users to user?

**Review by Gianni-Nicolas — CHANGES_REQUESTED** [review]

File: src/main/java/wolox/training/models/Book.java (hunk 1 above)
Comment (alanlamaswolox) [reply src/main/java/wolox/training/models/Book.java]: I couldn't make a attachment with the previous name and i realized that this was the problem

**Review by Gianni-Nicolas — APPROVED** [review]

**Review by AleFalcon — APPROVED** [review]
Great Job
